src/SDKs/_metadata/Azure-Kusto_resource-manager.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: latest
+AutoRest installed successfully.
```

Comment:
Was this file generated during code generation? Please delete this file and regenerate the code.